client/my-sites/controller.js

```diff
@@ -428,16 +428,16 @@ export function noSite( context, next ) {
 	return next();
 }
 
-export async function setExperimentVariation( context, next ) {
-	const experimentAssignment = await loadExperimentAssignment(
-		'calypso_sidebar_upsell_dedicated_upgrade_flow_202301'
-	);
+// export async function setExperimentVariation( context, next ) {
```

Comment:
I'd suggest not committing commented code like this - we can always use code that's historically available by looking at older versions of the code and PRs.

Reply:
Thanks for the reminder, I agree with you, I was rushing myself to put something out in code to share my thoughts 😅